scania-digital-design-system/sdds
# fix: math.div and browserlist update

**Describe pull-request**  
Fix for issues shown in the console during the build and publish process.

**Solving issue**  
No ticket was assigned to this one.

**How to test**  
1. Run build or start command
2. See if errors from below still show up


**Additional context**  
Errors like:

`DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.`

```
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
```
